Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
If you want to review the func names I would say let's keep them as they are and do this exercise then.. in fact we already discussed about grouping all the into a new `Movement` object or we could use the existing `Nozzle`.. 

`Nozzle::move_to_x()`, `Nozzle::move_to_axis()` and so on.

PS: Or maybe rename `Nozzle` to `Tool`.. dunno.